sonar-plugin-api/src/main/java/org/sonar/api/batch/postjob/PostJobContext.java

```diff
@@ -36,12 +37,14 @@
   // ----------- Only available in preview mode --------------
 
   /**
-   * All the unresolved issues of the project, including the issues reported by end-users.
+   * All the unresolved issues of the project, including the issues reported by end-users. Only available in preview/issues mode.
+   * @throw {@link UnsupportedOperationException} if not in preview/issues mode. To test the mode you can inject {@link AnalysisMode} in your constructor.
```

Comment:
what about adding method `analysisMode()` to this class as it will be frequently needed ?